### PR TITLE
Next Channel -> Canary Channel

### PR DESCRIPTION
**Pull request by acdlite**
See https://github.com/facebook/react/pull/26761 for more context.

### CODE REVIEW

File: src/content/community/versioning-policy.md [hunk 1]
```diff
@@ -80,12 +80,12 @@ This section will be most relevant to developers who work on frameworks, librari
...
+- [**Canary**](#canary-channel) tracks the main branch of the React source code repository. Think of these as release candidates for the next minor semver release. Use this for integration testing between React and third party projects.
 - [**Experimental**](#experimental-channel) includes experimental APIs and features that aren't available in the stable releases. These also track the main branch, but with additional feature flags turned on. Use this to try out upcoming features before they are released.
 
-All releases are published to npm, but only Latest uses semantic versioning. Prereleases (those in the Next and Experimental channels) have versions generated from a hash of their contents and the commit date, e.g. `0.0.0-68053d940-20210623` for Next and `0.0.0-experimental-68053d940-20210623` for Experimental.
+All releases are published to npm, but only Latest uses semantic versioning. Prereleases (those in the Canary and Experimental channels) have versions generated from a hash of their contents and the commit date, e.g. `18.3.0-canary-388686f29-20230503` for Next and `0.0.0-experimental-388686f29-20230503` for Experimental.
```
Comment (gaearon) [inline src/content/community/versioning-policy.md:86]: "for Next"

File: src/content/community/versioning-policy.md [hunk 2]
```diff
@@ -80,12 +80,12 @@ This section will be most relevant to developers who work on frameworks, librari
...
 
-**The only officially supported release channel for user-facing applications is Latest**. Next and Experimental releases are provided for testing purposes only, and we provide no guarantees that behavior won't change between releases. They do not follow the semver protocol that we use for releases from Latest.
+**The only officially supported release channel for user-facing applications is Latest**. Canary and Experimental releases are provided for testing purposes only, and we provide no guarantees that behavior won't change between releases. They do not follow the semver protocol that we use for releases from Latest.
```
Comment (gaearon) [inline src/content/community/versioning-policy.md:88]: This seems no longer true

Comment (gaearon) [reply src/content/community/versioning-policy.md:88]: Or do you want to do this separately

Comment (acdlite) [reply src/content/community/versioning-policy.md:88]: Yeah let's wait until the post is ready?

**Pull request merged**